packages/rest/test/integration/rest.application.integration.ts

```diff
@@ -71,6 +80,19 @@ describe('RestApplication (integration)', () => {
       .expect(content);
   });
 
+  it('gives precedence to API controllers over static assets', async () => {
```

Comment:
@bajtos here is the test which catches the problem. Calling `restApp.static('/html', root);` after  `restApp.controller(DummyController);` passes the test. That means, something's wrong in the order of registration /look up.